test/helpers/auto-fix-func.js

```diff
@@ -13,8 +13,10 @@ function autoFixFunc(node, validation, root, config) {
         // auto-fix by PostCSS AST tranformation
         // eslint-disable-next-line no-param-reassign
         node.value = '$color-red';
+        return;
     }
   }
+  throw `Property ${prop} with value ${value} can't be autofix (scale-unlimited/declaration-strict-value)`
```

Comment:
`(scale-unlimited/declaration-strict-value)` should not be part of the error.
The plugin adds it's name

Reply:
@devTeaa 
Readme needs to be updated to document that feature
https://github.com/AndyOGo/stylelint-declaration-strict-value#autofix-support

Reply:
I just submitted the changes, please help re-review it, thanks @AndyOGo